src/example.js

```diff
@@ -1,15 +1,16 @@
 /* eslint-env node, browser */
 'use strict';
 
-var analytics = require('./analytics.js');
-var sampleData = require('./sampledata.js');
+// require('./index.js');
+var analytics = hyperAnalytics;//eslint-disable-line no-undef
```

Comment:
preferred is to put `/* globals hyperAnalytics */` at top _or_ use `window.hyperAnalytics`